src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -63,7 +63,7 @@ public function exists($name)
      *
      * @throws \Twig_Error_Loader if the template could not be found
      */
-    protected function findTemplate($template)
+    protected function findTemplate($template, $throw = true)
     {
         $logicalName = (string) $template;
```

Comment:
We have a big problem here. In Twig 2.0, we don't convert template name to string anymore. But Symfony is made around this assumption. Failing tests are failing because they pass a `TemplateReference` instance directly to the `Twig_Loader_Array` class, which cannot deal with that anymore.

Moreover, if we were to refactor the Filesystem loader class to use composition, or just implement the interface and use the chain, that won't work as well for the same reason.

The only option I can see is to revert the non-string conversion in Twig 2.0 to keep BC.

What do you think? @stof @Tobion